<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EternalModManagerClass</class>
 <widget class="QMainWindow" name="EternalModManagerClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>796</width>
    <height>611</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Eternal mod manager</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="mainTabWidget">
      <property name="font">
       <font>
        <family>Yu Gothic UI</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="pluginsTab">
       <attribute name="title">
        <string>Plugins</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="1" column="3">
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QLabel" name="disabledPluginsLabel">
            <property name="text">
             <string>Disabled plugins</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="disabledPluginsAmountLabel">
            <property name="text">
             <string>NULL</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="1">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="totalPluginsLabel">
            <property name="text">
             <string>Total plugins:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="pluginsAmountLabel">
            <property name="text">
             <string>NULL</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="0" column="1" colspan="5">
         <widget class="QListWidget" name="pluginListWidget">
          <property name="font">
           <font>
            <family>Yu Gothic UI</family>
           </font>
          </property>
         </widget>
        </item>
        <item row="1" column="4">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="5">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="activePluginsLabel">
            <property name="text">
             <string>Active plugins:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="activePlguinsAmountLabel">
            <property name="text">
             <string>NULL</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QToolButton" name="movingPluginUpToolButton">
            <property name="text">
             <string>UP</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="movingPluginDownToolButton">
            <property name="text">
             <string>DOWN</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="disablingAllPluginsToolButton">
            <property name="text">
             <string>DISABLE</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="inclusionAllPluginsToolButton">
            <property name="text">
             <string>ENABLE</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="recountPluginsToolButton">
            <property name="text">
             <string>RECOUNT</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="uploadFileDownloadOrderTtoolButton">
            <property name="text">
             <string>LOAD</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="saveLoadingOrderFileToolButton">
            <property name="text">
             <string>SAVE</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="rescanGameToolButton">
            <property name="text">
             <string>RES</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="gameSelectToolButton">
            <property name="text">
             <string>GAME</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="gameRunToolButton">
            <property name="text">
             <string>EXE</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="aboutToolButton">
            <property name="text">
             <string>ABOUT</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item row="1" column="2">
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="modsTab">
       <attribute name="title">
        <string>Mods</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="1" column="1">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="totalModsLabel">
            <property name="text">
             <string>Total mods:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="totalModsAmountLabel">
            <property name="text">
             <string>NULL</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="3">
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="installedModsLabel">
            <property name="text">
             <string>Installed mods:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="installedModsAmountLabel">
            <property name="text">
             <string>NULL</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="2">
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="5">
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="disabledModsLabel">
            <property name="text">
             <string>Disabled mods:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="disabledModsAmountLabel">
            <property name="text">
             <string>NULL</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="0" column="0" rowspan="2">
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QToolButton" name="addingpluginToolButton">
            <property name="text">
             <string>ADD</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item row="0" column="1" colspan="5">
         <widget class="QListWidget" name="modsListWidget">
          <property name="font">
           <font>
            <family>Yu Gothic UI</family>
           </font>
          </property>
         </widget>
        </item>
        <item row="1" column="4">
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="settingsTab">
       <attribute name="title">
        <string>Settings</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_8">
        <item row="0" column="0">
         <widget class="QGroupBox" name="selectLanugageGroupBox">
          <property name="title">
           <string>Lanugage</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="0" column="0">
            <widget class="QComboBox" name="selectLanugageComboBox">
             <item>
              <property name="text">
               <string>English</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Russian</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QGroupBox" name="launchTypeGroupBox">
          <property name="title">
           <string>Launch type</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_5">
           <item row="0" column="0">
            <widget class="QComboBox" name="selectLaunchTypeComboBox">
             <item>
              <property name="text">
               <string>Launcher</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>EXE</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>SKSE</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QGroupBox" name="themeGroupBox">
          <property name="title">
           <string>Theme</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_6">
           <item row="0" column="0">
            <widget class="QComboBox" name="themeComboBox">
             <item>
              <property name="text">
               <string>Default</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Dark</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QGroupBox" name="generalGroupBox">
          <property name="title">
           <string>General</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_7">
           <item row="0" column="0">
            <widget class="QCheckBox" name="closeProgramAfterLaunchGameCheckBox">
             <property name="text">
              <string>Close program after launch game</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="4">
         <spacer name="horizontalSpacer_6">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="2">
         <spacer name="verticalSpacer_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="2" column="3" colspan="2">
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <spacer name="horizontalSpacer_5">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="applySettingsPushButton">
            <property name="text">
             <string>Apply</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="defaultSettingsPushButton">
            <property name="text">
             <string>Default</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>796</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="EternalModManager.qrc"/>
 </resources>
 <connections/>
</ui>
